### PR TITLE
Add lab_status.yaml

### LINKED ISSUE #19: Schema for lab status (shown on website)
This would substitute the `~staff/lab_status.yaml` file we currently use, which displays urgent announcements on our website. This one is probably easier than the other ones since we probably don't need to make ocflib functions for it if the code to parse the file already exists in ocfweb. Just needs a schema and a small change in ocfweb.

### PR DESCRIPTION
**Pull request by redplanks**
Fixes #19 

After this is merged, will test and submit the corresponding PR to ocfweb to read from this file.

### CODE REVIEW

**Comment by redplanks** [comment]
Maybe we should move the HTML to a template within ocfweb and have the yaml specify just its contents

**Comment by dkess** [comment]
We should be able to get this working without depending on ocfweb deploys.

**Comment by redplanks** [comment]
If puppet fails we can always manually pull/edit `/etc/ocf` on the k8s masters too

File: configs/lab_status.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Once this file is changed, you'll need to trigger a Puppet run on the Puppet master and then on the
```
Comment (dkess) [inline configs/lab_status.yaml]: Jenkins will automatically trigger a Puppet run on the Puppet master

**Comment by abizer** [comment]
sgtm

**Pull request merged**